### PR TITLE
Improve string localization

### PR DESCRIPTION
**Pull request by dasuni-30**
Screenshot 2023-11-09 at 12 34 59

### CODE REVIEW

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 1]
```diff
@@ -10197,8 +10197,9 @@ export const console: ConsoleNS = {
                                 label: "Assigned application",
                                 placeholder: "Select application to assign the role",
                                 applicationSubTitle: {
-                                    application: "Support application-scoped roles. ",
-                                    organization: "Support organization-scoped roles"
+                                    application: "Support application-scoped roles.",
+                                    organization: "Support organization-scoped role. ",
```
Comment (Rashmini) [inline modules/i18n/src/translations/en-US/portals/console.ts:10201]:
```suggestion
                                    organization: "Support organization-scoped role.",
```

File: modules/i18n/src/translations/en-US/portals/console.ts [hunk 2]
```diff
@@ -10197,8 +10197,9 @@ export const console: ConsoleNS = {
...
+                                    changeAudience: "Change the audience"
```
Comment (Rashmini) [inline modules/i18n/src/translations/en-US/portals/console.ts:10202]:
```suggestion
                                    changeAudience: "Change the audience."
```

**Review by Rashmini — APPROVED** [review]

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 1 above)
Comment (dasuni-30) [reply modules/i18n/src/translations/en-US/portals/console.ts:10201]: This spacing is intentional.

File: modules/i18n/src/translations/en-US/portals/console.ts (hunk 2 above)
Comment (dasuni-30) [reply modules/i18n/src/translations/en-US/portals/console.ts:10202]: This is a link text which does not require `.`. WDYT?

**Pull request merged**